Source/WebCore/editing/cocoa/DictionaryLookup.mm

```diff
@@ -313,7 +313,11 @@ static bool canCreateRevealItems()
     if (position.isNull())
         position = firstPositionInOrBeforeNode(node);
 
-    auto selection = CheckedRef(frame->page()->focusController())->focusedOrMainFrame().selection().selection();
+    RefPtr focusedOrMainFrame = CheckedRef(frame->page()->focusController())->focusedOrMainFrame();
```

Comment:
`frame->page()->checkedFocusController()->focusedOrMainFrame()` is what it's supposed to be. This code is a bit outdated and it'd be nice to refresh it.